polkadot/runtime/westend/src/lib.rs

```diff
@@ -1188,6 +1188,7 @@ impl parachains_scheduler::Config for Runtime {
 
 parameter_types! {
 	pub const BrokerId: u32 = BROKER_ID;
+	pub MaxXCMTransactWeight: Weight = Weight::from_parts(200_000_000, 20_000);
```

Comment:
Set lease has three reads and a write, independent of the ref time of the actual function on that runtime that already means a `ref_time` of `175_000_000`. Is it worth leaving a bit more of a buffer to account for fluctuations in benchmarking on top of this?